packages/playwright/src/common/ipc.ts

```diff
@@ -32,6 +32,7 @@ export type ConfigCLIOverrides = {
   reporter?: ReporterDescription[];
   additionalReporters?: ReporterDescription[];
   shard?: { current: number, total: number };
+  shardingSeed?: string;
```

Comment:
`shardingSeed` is required on `ConfigCLIOverrides` to support `--sharding-seed` cli option... It would be limiting to only support this via config file. Or am I missing something?

Reply:
…OR if not via cli option, it would be great to support it via environment variable e.g.`export PW_SHARDING_SEED=...`

Reply:
Ah, you are right, we need it here since we are setting it in the command line.